ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/ClaimMigrator.java

```diff
@@ -46,13 +49,27 @@ public void migrate() {
         AtomicInteger updatedClaims = new AtomicInteger(0);
         AtomicInteger failedMigrations = new AtomicInteger(0);
 
-        notMigratedClaims.parallelStream().forEach(claim -> {
-            migrationHandler.migrateClaim(migratedClaims, failedMigrations, updatedClaims, claim, user);
-        });
+        ForkJoinPool forkJoinPool = new ForkJoinPool(10);
+
+        try {
+            forkJoinPool.submit(() -> migrateClaims(user, notMigratedClaims, migratedClaims, updatedClaims, failedMigrations))
```

Comment:
how is this different from calling migrateClaims directly? Already the parallelstream is using forkjoinpool. If we are submitting a single runnable task, what advantage does this have?